### ISSUE TITLE
Exported deck from Anki Desktop, show card review finished and not imported when not using new schema in AnkiDroid

### ISSUE THREAD

**Issue by krmanik** [ankidroid/Anki-Android#12477]
###### Reproduction Steps

1. Create a deck on Anki Desktop and export
2. Copy the deck and import it in AnkiDroid 

###### Expected Result

It should show appropriate message for errors or import successfully. 

###### Actual Result

a) For new backend and not toggle on v3 scheduler, it is showing card review finished
![image](https://user-images.githubusercontent.com/12841290/191591261-67990b8f-34d4-4b1a-ba7d-bcec86bef290.png)
    
b) For both new backend and v3 toggle off, the is not imported and showing error
![image](https://user-images.githubusercontent.com/12841290/191591073-3efb2617-c9bc-4a9e-8d3d-5db4e9c27ecb.png)

###### Debug info
Refer to the [support page](https://ankidroid.org/docs/help.html) if you are unsure where to get the "debug info".
```
AnkiDroid Version = 2.16alpha85

Android Version = 11

Manufacturer = samsung

Model = SM-M307F

Hardware = exynos9611

Webview User Agent = Mozilla/5.0 (Linux; Android 11; SM-M307F Build/RP1A.200720.012; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/105.0.5195.136 Mobile Safari/537.36

ACRA UUID = 41b8bee7-c609-4622-b55f-6fa400bc4809

New schema = false

Scheduler = std2

Crash Reports Enabled = true

DatabaseV2 Enabled = true

```

```
AnkiDroid Version = 2.16alpha84

Android Version = 11

Manufacturer = samsung

Model = SM-M307F

Hardware = exynos9611

Webview User Agent = Mozilla/5.0 (Linux; Android 11; SM-M307F Build/RP1A.200720.012; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/105.0.5195.136 Mobile Safari/537.36

ACRA UUID = 79fe9b08-fde8-4674-9bee-b7724d606b6e

New schema = true

Scheduler = std2

Crash Reports Enabled = false

DatabaseV2 Enabled = true

```
###### Research
*Enter an [x] character to confirm the points below:*

- [x] I have read the [support page](https://ankidroid.org/docs/help.html) and am reporting a bug or enhancement request specific to AnkiDroid
- [x] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [x] I have searched for similar existing issues here and on the user forum
- [ ] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

**Comment by pavreh** [comment]
Import issue with the legacy schema is described in #11845.

**Comment by dae** [comment]
I can't reproduce the new schema issue: I create a deck with two new cards and export an .apkg, copy it to Android and import it with the new schema active, and I can tap on the deck to review the cards, both in v2 and v3.

**Comment by krmanik** [comment]
May be this is device specific errors. I will put the logcat in next comments.

1. Importing deck using new backend and v3 off

https://user-images.githubusercontent.com/12841290/191717277-624e0776-eb8d-4093-9839-259d995ef91f.mp4

2. Importing and reviewing when new v3 off

https://user-images.githubusercontent.com/12841290/191717211-231a04f5-0f62-4fc3-bc40-21b98f9ccf30.mp4

**Comment by krmanik** [comment]
Deck containing media is not importing.

This is logcat
```
2022-09-22 15:38:34.723 29190-31218 Collection              com.ichi2.anki                       I  (Re)opening Database: /storage/emulated/0/AnkiDroid/tmpzip/collection.anki2
2022-09-22 15:38:34.870 29190-31218 Collection              com.ichi2.anki                       I  Opening Collection Log
2022-09-22 15:38:34.887 29190-31218 Anki2Importer           com.ichi2.anki                       I  Preparing Import
2022-09-22 15:38:34.912 29190-31218 Anki2Importer           com.ichi2.anki                       I  Importing notes
2022-09-22 15:38:34.922 29190-31218 Anki2Importer           com.ichi2.anki                       W  java.io.FileNotFoundException: /storage/emulated/0/AnkiDroid/collection.media/cmn-我.mp3: open failed: ENOENT (No such file or directory)
                                                                                                    	at libcore.io.IoBridge.open(IoBridge.java:492)
                                                                                                    	at java.io.FileInputStream.<init>(FileInputStream.java:160)
                                                                                                    	at java.io.FileInputStream.<init>(FileInputStream.java:115)
                                                                                                    	at com.ichi2.libanki.importer.Anki2Importer._mediaData(Anki2Importer.kt:701)
                                                                                                    	at com.ichi2.libanki.importer.Anki2Importer._dstMediaData(Anki2Importer.kt:719)
                                                                                                    	at com.ichi2.libanki.importer.Anki2Importer._mungeMedia(Anki2Importer.kt:753)
                                                                                                    	at com.ichi2.libanki.importer.Anki2Importer._importNotes(Anki2Importer.kt:236)
                                                                                                    	at com.ichi2.libanki.importer.Anki2Importer._import(Anki2Importer.kt:116)
                                                                                                    	at com.ichi2.libanki.importer.Anki2Importer.run(Anki2Importer.kt:79)
                                                                                                    	at com.ichi2.libanki.importer.AnkiPackageImporter.run(AnkiPackageImporter.kt:142)
                                                                                                    	at com.ichi2.async.CollectionTask$ImportAdd.task(CollectionTask.kt:627)
                                                                                                    	at com.ichi2.async.CollectionTask$ImportAdd.task(CollectionTask.kt:613)
                                                                                                    	at com.ichi2.async.TaskDelegate.execTask(TaskDelegate.kt:50)
                                                                                                    	at com.ichi2.async.CollectionTask.actualDoInBackground(CollectionTask.kt:133)
                                                                                                    	at com.ichi2.async.CollectionTask.doInBackground(CollectionTask.kt:95)
                                                                                                    	at com.ichi2.async.CollectionTask.doInBackground(CollectionTask.kt:61)
                                                                                                    	at android.os.AsyncTask$3.call(AsyncTask.java:394)
                                                                                                    	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
                                                                                                    	at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:305)
                                                                                                    	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
                                                                                                    	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
                                                                                                    	at java.lang.Thread.run(Thread.java:923)
                                                                                                    Caused by: android.system.ErrnoException: open failed: ENOENT (No such file or directory)
                                                                                                    	at libcore.io.Linux.open(Native Method)
                                                                                                    	at libcore.io.ForwardingOs.open(ForwardingOs.java:166)
                                                                                                    	at libcore.io.BlockGuardOs.open(BlockGuardOs.java:254)
                                                                                                    	at libcore.io.ForwardingOs.open(ForwardingOs.java:166)
                                                                                                    	at android.app.ActivityThread$AndroidOs.open(ActivityThread.java:8494)
                                                                                                    	at libcore.io.IoBridge.open(IoBridge.java:478)
                                                                                                    	at java.io.FileInputStream.<init>(FileInputStream.java:160) 
                                                                                                    	at java.io.FileInputStream.<init>(FileInputStream.java:115) 
                                                                                                    	at com.ichi2.libanki.importer.Anki2Importer._mediaData(Anki2Importer.kt:701) 
                                                                                                    	at com.ichi2.libanki.importer.Anki2Importer._dstMediaData(Anki2Importer.kt:719) 
                                                                                                    	at com.ichi2.libanki.importer.Anki2Importer._mungeMedia(Anki2Importer.kt:753) 
                                                                                                    	at com.ichi2.libanki.importer.Anki2Importer._importNotes(Anki2Importer.kt:236) 
                                                                                                    	at com.ichi2.libanki.importer.Anki2Importer._import(Anki2Importer.kt:116) 
                                                                                                    	at com.ichi2.libanki.importer.Anki2Importer.run(Anki2Importer.kt:79) 
                                                                                                    	at com.ichi2.libanki.importer.AnkiPackageImporter.run(AnkiPackageImporter.kt:142) 
                                                                                                    	at com.ichi2.async.CollectionTask$ImportAdd.task(CollectionTask.kt:627) 
                                                                                                    	at com.ichi2.async.CollectionTask$ImportAdd.task(CollectionTask.kt:613) 
                                                                                                    	at com.ichi2.async.TaskDelegate.execTask(TaskDelegate.kt:50) 
                                                                                                    	at com.ichi2.async.CollectionTask.actualDoInBackground(CollectionTask.kt:133) 
                                                                                                    	at com.ichi2.async.CollectionTask.doInBackground(CollectionTask.kt:95) 
                                                                                                    	at com.ichi2.async.CollectionTask.doInBackground(CollectionTask.kt:61) 
                                                                                                    	at android.os.AsyncTask$3.call(AsyncTask.java:394) 
                                                                                                    	at java.util.concurrent.FutureTask.run(FutureTask.java:266) 
                                                                                                    	at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:305) 
                                                                                                    	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167) 
                                                                                                    	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641) 
                                                                                                    	at java.lang.Thread.run(Thread.java:923) 
2022-09-22 15:38:34.925 29190-31218 Anki2Importer           com.ichi2.anki                       E  _import() exception
                                                                                                    java.lang.NullPointerException
                                                                                                    	at com.ichi2.libanki.importer.Anki2Importer._mungeMedia(Anki2Importer.kt:776)
                                                                                                    	at com.ichi2.libanki.importer.Anki2Importer._importNotes(Anki2Importer.kt:236)
                                                                                                    	at com.ichi2.libanki.importer.Anki2Importer._import(Anki2Importer.kt:116)
                                                                                                    	at com.ichi2.libanki.importer.Anki2Importer.run(Anki2Importer.kt:79)
                                                                                                    	at com.ichi2.libanki.importer.AnkiPackageImporter.run(AnkiPackageImporter.kt:142)
                                                                                                    	at com.ichi2.async.CollectionTask$ImportAdd.task(CollectionTask.kt:627)
                                                                                                    	at com.ichi2.async.CollectionTask$ImportAdd.task(CollectionTask.kt:613)
                                                                                                    	at com.ichi2.async.TaskDelegate.execTask(TaskDelegate.kt:50)
                                                                                                    	at com.ichi2.async.CollectionTask.actualDoInBackground(CollectionTask.kt:133)
                                                                                                    	at com.ichi2.async.CollectionTask.doInBackground(CollectionTask.kt:95)
                                                                                                    	at com.ichi2.async.CollectionTask.doInBackground(CollectionTask.kt:61)
                                                                                                    	at android.os.AsyncTask$3.call(AsyncTask.java:394)
                                                                                                    	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
                                                                                                    	at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:305)
                                                                                                    	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
                                                                                                    	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
                                                                                                    	at java.lang.Thread.run(Thread.java:923)
2022-09-22 15:38:34.927 29190-31218 DB$Companion            com.ichi2.anki                       W  Not in a transaction. Cannot end transaction.
2022-09-22 15:38:34.929 29190-31218 DB                      com.ichi2.anki                       D  Database /storage/emulated/0/AnkiDroid/tmpzip/collection.media.ad.db2 closed = true
2022-09-22 15:38:34.930 29190-31218 Collection              com.ichi2.anki                       I  Closing Collection Log
2022-09-22 15:38:34.930 29190-31218 Collection              com.ichi2.anki                       I  Collection closed
2022-09-22 15:38:34.931 29190-31218 Anki2Importer           com.ichi2.anki                       E  Exception while importing
                                                                                                    java.lang.NullPointerException
                                                                                                    	at com.ichi2.libanki.importer.Anki2Importer._mungeMedia(Anki2Importer.kt:776)
                                                                                                    	at com.ichi2.libanki.importer.Anki2Importer._importNotes(Anki2Importer.kt:236)
                                                                                                    	at com.ichi2.libanki.importer.Anki2Importer._import(Anki2Importer.kt:116)
                                                                                                    	at com.ichi2.libanki.importer.Anki2Importer.run(Anki2Importer.kt:79)
                                                                                                    	at com.ichi2.libanki.importer.AnkiPackageImporter.run(AnkiPackageImporter.kt:142)
                                                                                                    	at com.ichi2.async.CollectionTask$ImportAdd.task(CollectionTask.kt:627)
                                                                                                    	at com.ichi2.async.CollectionTask$ImportAdd.task(CollectionTask.kt:613)
                                                                                                    	at com.ichi2.async.TaskDelegate.execTask(TaskDelegate.kt:50)
                                                                                                    	at com.ichi2.async.CollectionTask.actualDoInBackground(CollectionTask.kt:133)
                                                                                                    	at com.ichi2.async.CollectionTask.doInBackground(CollectionTask.kt:95)
                                                                                                    	at com.ichi2.async.CollectionTask.doInBackground(CollectionTask.kt:61)
                                                                                                    	at android.os.AsyncTask$3.call(AsyncTask.java:394)
                                                                                                    	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
                                                                                                    	at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:305)
                                                                                                    	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
                                                                                                    	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
                                                                                                    	at java.lang.Thread.run(Thread.java:923)
2022-09-22 15:38:34.932 29190-31218 AnkiPackageImporter     com.ichi2.anki                       D  Total available size is: 35987357696
2022-09-22 15:38:35.197 29190-31218 Collection...$ImportAdd com.ichi2.anki                       W  com.ichi2.anki.exception.ImportExportException
                                                                                                    	at com.ichi2.libanki.importer.Anki2Importer.run(Anki2Importer.kt:85)
                                                                                                    	at com.ichi2.libanki.importer.AnkiPackageImporter.run(AnkiPackageImporter.kt:142)
                                                                                                    	at com.ichi2.async.CollectionTask$ImportAdd.task(CollectionTask.kt:627)
                                                                                                    	at com.ichi2.async.CollectionTask$ImportAdd.task(CollectionTask.kt:613)
                                                                                                    	at com.ichi2.async.TaskDelegate.execTask(TaskDelegate.kt:50)
                                                                                                    	at com.ichi2.async.CollectionTask.actualDoInBackground(CollectionTask.kt:133)
                                                                                                    	at com.ichi2.async.CollectionTask.doInBackground(CollectionTask.kt:95)
                                                                                                    	at com.ichi2.async.CollectionTask.doInBackground(CollectionTask.kt:61)
                                                                                                    	at android.os.AsyncTask$3.call(AsyncTask.java:394)
                                                                                                    	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
                                                                                                    	at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:305)
                                                                                                    	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
                                                                                                    	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
                                                                                                    	at java.lang.Thread.run(Thread.java:923)
2022-09-22 15:38:35.206 29190-31188 libEGL                  com.ichi2.anki                       W  EGLNativeWindowType 0x782fb88010 disconnect failed
```

**Comment by dae** [comment]
Are you able to share the .apkg for the second issue?

**Comment by krmanik** [comment]
Sure, The zip file contains the deck. The deck does not contain audio file for the words.
[Test Deck.zip](https://github.com/ankidroid/Anki-Android/files/9630882/Test.Deck.zip)

**Comment by dae** [comment]
Still can't reproduce :-( I imported that into new schema+no v3, tapped on Test, and it showed me a card.

**Comment by krmanik** [comment]
Okay, I am again check the exact steps for it and update here.

**Comment by krmanik** [comment]
I think there should card template errors because after some changes to card templates I also did not get the errors. This deck causing `review finished` bug.

[review finished bug.zip](https://github.com/ankidroid/Anki-Android/files/9630994/review.finished.bug.zip)

Another exception when subdeck exported and imported, wit both v3 and new schema off.
[new__test deck.zip](https://github.com/ankidroid/Anki-Android/files/9630963/new__test.deck.zip)

```
2022-09-23 11:12:44.966 21049-21667 Utils                   com.ichi2.anki                       D  Utils.writeToFile: Size: 1004 Kb, Duration: 0 s, Speed: 6236 Kb/s
2022-09-23 11:12:44.968 21049-21667 Utils                   com.ichi2.anki                       I  uncompress media
2022-09-23 11:12:44.969 21049-21667 Utils                   com.ichi2.anki                       D  Creating new file... = /storage/emulated/0/AnkiDroid/tmpzip/media
2022-09-23 11:12:45.028 21049-21667 Utils                   com.ichi2.anki                       D  Finished writeToFile!
2022-09-23 11:12:45.029 21049-21667 Utils                   com.ichi2.anki                       D  Utils.writeToFile: Size: 0 Kb, Duration: 0 s, Speed: 0 Kb/s
2022-09-23 11:12:45.030 21049-21667 Backend                 com.ichi2.anki                       D  Opening rust backend with lang=[en-IN]
2022-09-23 11:12:45.041 21049-21667 Collection              com.ichi2.anki                       I  (Re)opening Database: /storage/emulated/0/AnkiDroid/tmpzip/collection.anki2
2022-09-23 11:12:45.440 21049-21667 Collection              com.ichi2.anki                       I  Opening Collection Log
2022-09-23 11:12:45.470 21049-21667 AnkiPackageImporter     com.ichi2.anki                       D  Total available size is: 18963238912
2022-09-23 11:12:45.597 21049-21585 LeakCanary              com.ichi2.anki                       D  Setting up flushing for Thread[Filter,5,main]
2022-09-23 11:12:45.771 21049-21667 AndroidRuntime          com.ichi2.anki                       E  FATAL EXCEPTION: AsyncTask #1
                                                                                                    Process: com.ichi2.anki, PID: 21049
                                                                                                    java.lang.RuntimeException: An error occurred while executing doInBackground()
                                                                                                    	at android.os.AsyncTask$4.done(AsyncTask.java:415)
                                                                                                    	at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:383)
                                                                                                    	at java.util.concurrent.FutureTask.setException(FutureTask.java:252)
                                                                                                    	at java.util.concurrent.FutureTask.run(FutureTask.java:271)
                                                                                                    	at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:305)
                                                                                                    	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
                                                                                                    	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
                                                                                                    	at java.lang.Thread.run(Thread.java:923)
                                                                                                    Caused by: java.lang.NullPointerException
                                                                                                    	at com.ichi2.libanki.Decks.childMap(Decks.kt:909)
                                                                                                    	at com.ichi2.libanki.sched.SchedV2.unburyCardsForDeck(SchedV2.kt:2207)
                                                                                                    	at com.ichi2.libanki.sched.BaseSched.unburyCardsForDeck$default(BaseSched.kt:118)
                                                                                                    	at com.ichi2.libanki.sched.BaseSched.unburyCards(BaseSched.kt:143)
                                                                                                    	at com.ichi2.libanki.sched.SchedV2._updateCutoff$lambda-7(SchedV2.kt:1990)
                                                                                                    	at com.ichi2.libanki.sched.SchedV2.$r8$lambda$cYK-YCKvF7aJH0WnpwDtzp0qzm0(Unknown Source:0)
                                                                                                    	at com.ichi2.libanki.sched.SchedV2$$ExternalSyntheticLambda2.run(Unknown Source:2)
                                                                                                    	at com.ichi2.utils.SyncStatus$Companion.ignoreDatabaseModification(SyncStatus.kt:97)
                                                                                                    	at com.ichi2.libanki.sched.SchedV2._updateCutoff(SchedV2.kt:1990)
                                                                                                    	at com.ichi2.libanki.sched.SchedV2.<init>(SchedV2.kt:2684)
                                                                                                    	at com.ichi2.libanki.Collection._loadScheduler(Collection.kt:266)
                                                                                                    	at com.ichi2.libanki.Collection.<init>(Collection.kt:198)
                                                                                                    	at com.ichi2.libanki.Storage.collection(Storage.kt:88)
                                                                                                    	at com.ichi2.libanki.Storage.collection$default(Storage.kt:79)
                                                                                                    	at com.ichi2.libanki.importer.AnkiPackageImporter.run(AnkiPackageImporter.kt:101)
                                                                                                    	at com.ichi2.async.CollectionTask$ImportAdd.task(CollectionTask.kt:627)
                                                                                                    	at com.ichi2.async.CollectionTask$ImportAdd.task(CollectionTask.kt:613)
                                                                                                    	at com.ichi2.async.TaskDelegate.execTask(TaskDelegate.kt:50)
                                                                                                    	at com.ichi2.async.CollectionTask.actualDoInBackground(CollectionTask.kt:133)
                                                                                                    	at com.ichi2.async.CollectionTask.doInBackground(CollectionTask.kt:95)
                                                                                                    	at com.ichi2.async.CollectionTask.doInBackground(CollectionTask.kt:61)
                                                                                                    	at android.os.AsyncTask$3.call(AsyncTask.java:394)
                                                                                                    	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
                                                                                                    	at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:305) 
                                                                                                    	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167) 
                                                                                                    	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641) 
                                                                                                    	at java.lang.Thread.run(Thread.java:923) 
2022-09-23 11:12:45.774 21049-21667 UsageAnalytics          com.ichi2.anki                       D  sendAnalyticsException() description/fatal: java.lang.NullPointerException/true
2022-09-23 11:12:45.793 21049-21667 ACRA                    com.ichi2.anki                       I  ACRA is disabled for com.ichi2.anki - forwarding uncaught Exception on to default ExceptionHandler
2022-09-23 11:12:45.880 21049-21049 DeckPicker              com.ichi2.anki                       D  onPause()
```

**Comment by dae** [comment]
Thanks @krmanik, a fix for this will be in 2.1.55

**Comment by krmanik** [comment]
Thanks for the fix.
